documentation/PreferencesServer.md

```diff
@@ -349,6 +349,202 @@ There are two important things to note here:
    to `http://registry.gpii.net/common/fontSize`. Since we do not allow the same setting to be present multiple times in
    the NP set, the fontSize has been stored in the flat ontology and removed from the ISO24751 block.
 
+### GET /prefssafe/:prefsSafeId
+
+This end point will return the entire preferences safe, including the embedded preference sets. The URL param is the
+`id` of the preferences safe. This will not return any attached keys, credentials, or other records.
+
+### GET /prefssafe-with-keys/:prefsSafeId
+
+This endpoint will return the entire preferences safe structure, along with records directory related to it, such
+as keys and credentials. A top level object will contain the preferences safe under key `prefsSafe`, and the related
+documents under key `keys`.
+
+Example returned item:
+
+```json
+{
+    "prefsSafe": {
+        "id": "prefsSafeId",

```

Comment:
I think it might be more explanatory to use real data in examples. Also expand omitted blocks like line 371, 373. Let me know what you think. Thanks.